src/core/human_player.h

```diff
@@ -19,7 +19,7 @@ class HumanPlayer : public Player {
 
   _Action act(State& state) {
     int index = state.humanInputAction();
-    assert(false);
+    assert(false);  // <-- what??
```

Comment:
This means that this code should never run, and if it runs there's a bug. If true it means this function `act()` can be entirely eliminated because it's never called.

Reply:
Shall we try removing it?

Reply:
Yes, we should :P

Reply:
done

Reply:
Doesn't look done in the source view, did you push it?

Reply:
I did, I'm not sure why it's not appearing – https://github.com/AlignmentResearch/polygames/blob/fix_mcts_weirdness/src/core/human_player.h